src/app/Fake.FluentMigrator/Sample/CatsDatabase.cs

```diff
@@ -0,0 +1,38 @@
+using FluentMigrator;
+
+namespace Sample
```

Comment:
Should we announce that sample somewhere in the docs / API-Reference documentation?